.travis.yml

```diff
@@ -6,6 +6,7 @@ cache:
   directories:
     - $HOME/.composer/cache
     - downloads
+    - vendor
```

Comment:
i have found it in this blog post (https://mwop.net/blog/2015-12-14-secure-phar-automation.html) while i am writing on a private project. thought that it might be a good idea but yeah the improvements of speed is quite less to nothing i will remove it.